### ISSUE TITLE
Update of stable version, may be?

### ISSUE THREAD

**Issue by CitrusAlex** [YePpHa/YouTubeCenter#1421]
Hi. I'm user of Yandex.Browser, which is the fork of the chromium, but it's IMHO much better than Chrome. The Yandex.Browser allow you to use the Opera Store to install the extensions, so I can actually install the chromium version of YouTube Center. However, version of YT in the Opera Store IS SO DAMM GLITCHY! It's so crappy and outdated that I can't actually use it normally. So I have to the Developer Version from crx file, which is working PERFECT! Nevertheless, because of method of installing, every time I open the browser I have to enable the YT again and again and again. It's very annoying. So, how about to update the stable version of YT to the same version as Developer Version? For me it is working perfectly and DEFINITELY BETTER than stable version (what an irony).

**Comment by Yonezpt** [comment]
It is up to the author to decide wether or not the dev version is ready for being shipped into the stable version. Quite a few Chrome users have requested what you asked, also because of an annoying popup, but pushing stable versions just for the sake of people not being annoyed by their own browsers' annoyance defeats the whole purpose of the developer version, which is to test and provide feedback.

The Chrome users' best option was to start using YTC dev userscript with the Tampermonkey extension (I suspect it will work with Yandex.Browser because it is also a blink browser), I suggest you try the same. If you don't know how to install userscripts in Tampermonkey let me know and I will see how I can help.

**Comment by CitrusAlex** [comment]
Okay, but you must agree - the stable version is too outdated and because of it, glitchy. I hope the author will solve this problem and update the stable version of YT for chrome and other browsers.

**Comment by ReporterX** [comment]
I agree with the OP. The current stable version is not stable ironically speaking. New users may not look deep enough to try the developer build (which is much better now) before they get scared off. 

It is high time to find a stable version in the developer version (as the author deems) and upgrade to that version.

**Comment by ErdeFB** [comment]
Too bad that the developer version is way too unstable to be called stable. Therefore, I highly doubt that you will be getting a new stable version before some outstanding issues are addressed. No one is forcing you guys to suggest fixes or pay for someone to make them if you yourself cannot code. Or or just be patient and wait.

**Comment by ReporterX** [comment]
@ErdeFB Well the current developer build is stable to me, minus minor bugs and annoyances (I have written custom styles to fix them). I'm using Firefox.

Some say the latest 473 causes some major issues. 471 is better.

If the developer version is "way too unstalbe", the current so-called stable version is "way way way too unstable". Many bugs have been fixed since last "stable" version. IMHO a update is still preferable.

**Comment by ErdeFB** [comment]
Opinion noted. However, as the developer has not made a stable version yet, it would seem he does not agree with you.

Anyways, I once again would like to remind people that this is an issue tracker and not a discussion forum. Please keep this in mind. As this issue report does not benefit from debating, I'd suggest you guys should forward such concerns to the developer himself. You can find his contact information in his bio.
